### PR TITLE
BaSP-TG-107:  Admin add projects

### PR DESCRIPTION
**Pull request by Julian25**
Work donde on this branch:

- Generated the logic to add more than one employee when creating a project
- Developed the necessary logic so that the data of the admin who is logged in is automatically loaded when creating a project.

### CODE REVIEW

File: src/Components/Shared/Input/index.jsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
   return (
     <div className={type === 'checkbox' ? styles.check : styles.inputContainer}>
       <label htmlFor={id}>{text}</label>
-      <input type={type} name={id} {...register(id)} className={error && styles.inputError} />
+      <input
+        type={type}
+        name={id}
+        {...register(id)}
+        className={error && styles.inputError}
+        value={value}
```
Comment (lucasrendo) [inline src/Components/Shared/Input/index.jsx:13]: Is this necessary?

File: src/Components/Admin/CreateProject/index.jsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  const formatEmployees = () => {
+    return employeeList.map((employee) => {
+      return { id: employee._id, text: `${employee.firstName} ${employee.lastName}` };
+    });
+  };
+
+  const submitHandler = (data) => {
+    const project = data;
+    project.admin = adminData();
+    id ? dispatch(editProject(data, id)) : dispatch(addProject(data));
```
Comment (lucasrendo) [inline src/Components/Admin/CreateProject/index.jsx:105]: i think there's no need to edit projects from this screen

**Review by lucasrendo — COMMENTED** [review]
Features are great: clean and short code. One doubt, shouldn't be this component only for creating a project and no editing features?

**Review by lucasrendo — APPROVED** [review]

**Review by GonzaloBracamonte — APPROVED** [review]
Looks good!

**Review by MatiasJoaquinReynoso — APPROVED** [review]
All works correctly :)

**Pull request merged**